lib/cli/src/Cardano/CLI.hs

```diff
@@ -90,7 +90,7 @@ import Cardano.BM.Setup
 import Cardano.BM.Trace
     ( Trace, appendName, logAlert, logInfo )
 import Cardano.Wallet.Api
-    ( Api )
+    ( CoreApi )
```

Comment:
We don't provide any support for operations on Byron wallets in the CLI, so there's no need to import the whole API: we can just import the core.